app/src/main/java/org/schulcloud/mobile/controllers/main/MainFragment.kt

```diff
@@ -103,7 +103,7 @@ abstract class MainFragment<VM : ViewModel>(refreshableImpl: RefreshableImpl = R
             }
             R.id.base_action_refresh -> performRefresh()
         // TODO: Remove when deep linking is readded
-            R.id.base_action_openInBrowser -> context?.openUrl(url.asUri())
+            R.id.base_action_openInBrowser -> context?.openUrl(url.asPrefixedUri())
```

Comment:
Please also have a look at the share action in line 98 where this would be useful too. E.g., change `context.shareLink` to only accept URIs which are created using `asUri` (see other comment) and convert them back to a string.